app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,6 @@
 class BookingsController < ApplicationController
+  
+  def index
+    @bookings = Booking.joins(:resource).where(:bookings => {:user_id => current_user})
```

Comment:
Because we have `has_many :bookings` in User association, we can simply do
`current_user.bookings.joins(:resource)` but this also works!